
 This patch corrects a problem preventing multiple 
 simultaneous jobs from different pools.
 Apply to 1.36.2 with:

 cd <bacula-source>
 patch -p0 <1.36.2-pool.patch
 make
 ...

Index: src/dird/jobq.c
===================================================================
RCS file: /cvsroot/bacula/bacula/src/dird/jobq.c,v
retrieving revision 1.25.4.2
diff -u -r1.25.4.2 jobq.c
--- src/dird/jobq.c	15 Feb 2005 11:51:03 -0000	1.25.4.2
+++ src/dird/jobq.c	4 Mar 2005 13:16:19 -0000
@@ -584,7 +584,7 @@
 		   if (njcr->JobId == 0 || njcr == jcr) {
 		      continue;
 		   }
-		   if (njcr->pool != jcr->pool) {
+		   if (njcr->store == jcr->store && njcr->pool != jcr->pool) {
 		      skip_this_jcr = true;
 		      break;
 		   }
